### DISCUSSION TITLE
Kotlin target

### DISCUSSION THREAD

**Discussion by andre-santos-pt** [antlr/intellij-plugin-v4#608] [Q&A]
Hi,

I'd like to add support for a Kotlin target, namely using this: https://github.com/Strumenta/antlr-kotlin

How can I configure the plugin for this?

Thanks

**Comment by KvanTTT** [comment]
Hi. I'm afraid it's not possible because this Kotlin target is not officially supported (not placed to ANTLR core repository). Moreover, it's outdated.
